frontend/apps/reader/modules/readerlink.lua

```diff
@@ -720,6 +720,16 @@ function ReaderLink:onAddCurrentLocationToStack(show_notification)
     end
 end
 
+-- to be used on shortcuts for non-touch devices
+function ReaderLink:onAddCustomCurrentLocationToStack(show_notification)
```

Comment:
Good name doesn't need a comment, `onAddCurrentLocationToStackNonTouch`, for example.